src/App.js

```diff
@@ -88,9 +88,9 @@ export default function App() {
                 <Route path="/login" component={Login} />
                 <Route path="/forget" component={ForgetPassword} />
                 <Route path="/sign-up" component={Signup} />
-                <Route path="/mybeans" component={MyBeans} />
-                <AuthRoute path="/bean/:id" component={BeanEdit} />
-                <AuthRoute path="/bean" component={BeanCreate} />
+                <AuthRoute path="/bean/:id/edit" component={BeanEdit} />
+                <AuthRoute path="/bean/:id/create" component={BeanCreate} />
+                <Route path="/bean/:id" component={MyBeans} />
```

Comment:
これは

```suggestion
                <Route path="/beans" component={MyBeans} />
```

が望ましいと思います．
mybeansは豆のidは関係ないので．